apps/app/lib/api.ts

```diff
@@ -1,26 +1,32 @@
-import axios, { AxiosResponse } from "axios";
+import axios, { AxiosRequestConfig, AxiosRequestHeaders, AxiosResponse } from "axios";
 
 /**
  * Base URL builder for AKSARA.
  * @param base "api" | "local"
+ * @param {boolean} contentTypeHeader add Content-Type: "multipart/form-data" to header
  * @returns Base of URL
  *
  * @example "api"   -> "https://[NEXT_PUBLIC_API_URL]/"
  * @example "local" -> "https://[NEXT_PUBLIC_APP_URL]/"
  */
-const instance = (base: "api" | "local" | string = "api") => {
-  const baseURL =
-    base === "api"
-      ? process.env.NEXT_PUBLIC_API_URL
-      : base === "local"
-      ? process.env.NEXT_PUBLIC_APP_URL
-      : base;
-  return axios.create({
-    baseURL,
+const instance = (base: "api" | "local" | string = "api", contentTypeHeader?: boolean) => {
+  const config: AxiosRequestConfig = {
+    baseURL:
+      base === "api"
+        ? process.env.NEXT_PUBLIC_API_URL
+        : base === "local"
+        ? process.env.NEXT_PUBLIC_APP_URL
+        : base,
     headers: {
       Authorization: process.env.NEXT_PUBLIC_AUTHORIZATION_TOKEN,
     },
-  });
+  };
+  if (contentTypeHeader)
+    config.headers = {
+      ...config.headers,
+      ["Content-Type"]: "multipart/form-data",
+    };
+  return axios.create(config);
 };
```

Comment:
No, pls don't do boolean for `contentTypeHeader`. make the param flexible, in case we need to pass other type of headers. Refactor to something like this: 
```ts
// params: headers: Record<string, string> = {}
headers: {
      Authorization: process.env.NEXT_PUBLIC_AUTHORIZATION_TOKEN,
     ...headers
    },
```